### PR TITLE
Nadiaagahozo

### PR DESCRIPTION
**Pull request by nadiaagahozo**

### CODE REVIEW

File: app/controllers/api/capstones_controller.rb [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+  def create
+    @capstone = Capstone.new(
+      student_id: current_student.id,
+      name: params[:name],
+      description: params[:description],
+      url: params[:url],
+      screenshot_url: params[:screenshot_url]
+    )
+    if @capstone.save
+    render "show.json.jb"
```
Comment (thewendels) [inline app/controllers/api/capstones_controller.rb]: indent this line

File: app/controllers/api/skills_controller.rb [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+  def create
+    @skill = Skill.new(
+      student_id: current_student.id,
+      skill_name: params[:skill_name],
+    )
+    if @skill.save
+      render "show.json.jb"
+      else
+        render json: { errors: @skill.errors.full_messages }, status: :bad_request
+      end
```
Comment (thewendels) [inline app/controllers/api/skills_controller.rb:17]: these should move to the left to fall under "if"

File: app/views/api/capstones/_capstone.json.jb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  student_id: capstone.student.id,
```
Comment (pedronotpaydro) [inline app/views/api/capstones/_capstone.json.jb:2]: student_id: capstone.student_id,

File: app/views/api/skills/_skill.json.jb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  student_id: skill.student.id,
```
Comment (pedronotpaydro) [inline app/views/api/skills/_skill.json.jb:2]: student_id: skill.student_id,

**Pull request closed without merging**